com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md

```diff
@@ -151,6 +151,11 @@ Shader "Hidden/Shader/GrayScale"
         UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
         uint2 positionSS = input.texcoord * _ScreenSize.xy;
+
+        //NOTE: As of 12.x beta, for new custom post processes to be compatible with Dynamic resolution, please use:
+        //uint2 positionSS = input.texcoord * _PostProcessScreenSize.xy;
+        // This position must be used ultimately for color / normal and velocity. If not, your custom post process won't work when dynamic resolution / DLSS is enabled.
```

Comment:
Small adjustment: 

You must use this position for color / normal and velocity. Otherwise, your custom post-process won't work when dynamic resolution or DLSS is enabled.